src/js/now-value-service.js

```diff
@@ -0,0 +1,27 @@
+(function($){
```

Comment:
Once this service may admit more methods related to the USD value (not just to get the current one), prefer to name it as `usd-value-service.js` and let it expose the `getCurrentValue` method.